src/OrchardCore.Modules/OrchardCore.Facebook/Settings/FacebookLoginSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Facebook.Settings
+{
+    public class FacebookLoginSettings
+    {
+        public string CallbackPath { get; set; }
```

Comment:
Using `PathString` like we now do in the OpenID module could have been a good idea :smile: